connector/mqtt/vernemq/v2k-brigde/bin/scripts_tls/saveFormattedCRT.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BASE_DIR=${BASE_DIR:-"/v2k_bridge"}
+
+. "${BASE_DIR}"/bin/scripts_tls/_initVariables.sh
+
+_saveFormattedCRT()
+{
+  nameFile=$1
+  rawCRT=$2
```

Comment:
these variables should be marked as `local` to be visible only within the function